module/utils/useItems.js

```diff
@@ -0,0 +1,49 @@
+Hooks.on("renderChatMessage", (message, html, data) => {
+    setTimeout(() => {
+        $(html).find(".reduce-item-count").on("click", (event) => useItem(event));
+    }, 500);
+});
+
+export async function useItem(event){
+    //prevent the default action of the button
+    event.preventDefault();
+
+    // Get the item ID and name from the button element's data-item-id and data-item-name attributes
+    let itemId = event.currentTarget.dataset.item;
+    let itemName = event.currentTarget.dataset.itemName;
+    let parentId = `${event.currentTarget.dataset.itemParentid}`; //convert parentId to string
+    
+    //disable the button
+    event.currentTarget.disabled = true;
```

Comment:
Should this not be done by default and only if the quantity goes to <0?

Reply:
this is done so that the button isn't clicked twice and the amount reduced twice.